src/core/style/mixins.scss

```diff
@@ -114,6 +114,7 @@
 @mixin rtl-prop($ltr-prop, $rtl-prop, $value) {
   #{$ltr-prop}: $value;
   [dir=rtl] & {
+    #{$ltr-prop}: auto;
```

Comment:
As [initial](http://caniuse.com/#feat=css-initial-value) isn't supported on IE i think i might have to do something like:
```css
#{$ltr-prop}: auto;
#{$ltr-prop}: initial;
```